frontend/src/js/components/TicketCountSelector.tsx

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+import _ = require('underscore');
+import React = require('react');
+import Bootstrap = require('react-bootstrap');
+
+import Seat from './Seat.tsx';
+import {IShow} from "../../../../backend/src/show";
+import {ISection} from "../../../../backend/src/venue";
+import {IVenue} from "../../../../backend/src/venue";
+
+export interface ITicketCountSelectorProps {
```

Comment:
Would it make sense to have the same interface for both SeatSelector & TicketCountSelector props?